### PR TITLE
Add new script command: json to golang struct tool

### PR DESCRIPTION
**Pull request by tiancheng92**
## Description
convert the copied json into golang structure

## Type of change
- [x] New script command

## Screenshot
json_data: 
```json
{"status": {"message": "All Good!"}}
```

raw_data:
```
{"status": {"message": "All Good
```

## Dependencies / Requirements
* Dependency: requires jq, json-to-go-cli
* Install jq with Homebrew: brew install jq
* Install json-to-go-cli with npm: npm i json-to-go-cli -g

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/json-to-go-struct.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Dependency: requires json-to-go-cli, jq
+# Install with npm: `npm i json-to-go-cli -g`
+# Install with brew: `brew install jq`
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Json to go struct
```
Comment (dehesa) [inline commands/developer-utils/json-to-go-struct.sh:9]: Rename to: `JSON to Go Struct`

File: commands/developer-utils/json-to-go-struct.sh [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+# @raycast.packageName Developer Utilities
+
+# Optional parameters:
+# @raycast.argument1 {"type": "text", "placeholder": "inline", "optional": true}
+# @raycast.icon images/go.jpg
+
+# Documentation:
+# @raycast.author tiancheng92
+# @raycast.authorURL https://github.com/tiancheng92
+# @raycast.description convert the copied json into golang structure
```
Comment (dehesa) [inline commands/developer-utils/json-to-go-struct.sh:20]: Capitalize the first letter and add an ending dot.
`Convert the copied JSON into a golang structure.`

File: commands/developer-utils/json-to-go-struct.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+export LC_ALL=en_US.UTF-8
+export PATH="/opt/homebrew/bin:$PATH"
```
Comment (dehesa) [inline commands/developer-utils/json-to-go-struct.sh:23]: Avoid exporting if possible.

For example, in this case I am aware you are referencing the new homebrew path for M1 machines. However, this is a per-user configuration (not everyone has a M1 machine and not everyone uses brew). So it is much better if you remove this line and set this up for yourself in your home folder in `.zshrc` or `.zshenv` files.

**Review by dehesa — CHANGES_REQUESTED** [review]

File: commands/developer-utils/json-to-go-struct.sh (hunk 3 above)
Comment (dehesa) [reply commands/developer-utils/json-to-go-struct.sh:23]: I accepted your last PR without mentioning this. Could you please fix the translation shell scripts to and send a PR?

**Comment by dehesa** [comment]
Thanks for the PR @tiancheng92. Please check the comments and change the Go icon to be square and of 64x64 pixels.

**Comment by tiancheng92** [comment]
l have update json-to-go-struct.sh and replace image/go.png.

**Pull request merged**